bitbots_blackboard/src/bitbots_blackboard/capsules/world_model_capsule.py

```diff
@@ -363,6 +363,24 @@ def get_current_position(self):
         theta = euler_from_quaternion([orientation.x, orientation.y, orientation.z, orientation.w])[2]
         return transform.transform.translation.x, transform.transform.translation.y, theta
 
+    def get_current_position_pose_stamped(self):
```

Comment:
This could be somehow merged with `get_current_position`

Reply:
i refactored, can you check again?

Reply:
lgtm